src/Admin/Customizer/EmailCampaign/Customizer.php

```diff
@@ -368,12 +368,18 @@ public function save_email_campaign_title($wp_customize_manager)
     public function add_sections($wp_customize)
     {
         if ( ! apply_filters('mo_email_customizer_disable_upsell_section', false)) {
+
+            $capability = 'manage_options';
+            if( current_user_can('manage_mailoptin')){
```

Comment:
@picocodes instead of these multiple if statements, please create a global function at https://github.com/mailoptin/core/blob/master/src/Functions/functions.php that returns the capability string. the function should be like

```
function mo_capability() {
$capability = 'manage_options';
            if( current_user_can('manage_mailoptin')){
                $capability = 'manage_mailoptin';
            }

return $capability;
}
```

Also create another helper function that does `( current_user_can('administrator') || current_user_can('manage_mailoptin') )`

perhaps call it `mo_current_user_has_privilege` or similar name. Obviously the function won't accept an argument.

This will simplify the code don't you think?

Reply:
Brilliant. I don't know why I did not think of that before.